02-peer-protocol.md

```diff
@@ -787,9 +787,17 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 #### Requirements
 
 A sending node:
-  - MUST NOT offer `amount_msat` it cannot pay for in the
-remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+  - if it is _responsible_ for paying the Bitcoin fee:
+    - MUST NOT offer `amount_msat` it cannot pay for in the remote commitment
+    transaction at the current `feerate_per_kw` (see "Updating Fees") while
+    maintaining its channel reserve.
```

Comment:
Must also not offer an htlc if it cannot be paid for when the remote adds this htlc to the funder's local commitment tx. There can be a different dust limit and it therefore needs to be checked as well.

Reply:
True, this sentence also needs clarifying, it seems that lnd, c-lightning and eclair all understood it the same way but the spec isn't clear enough. I'll give a shot at clarifying that.

Reply:
Can we simply change `in the remote commitment transaction` to `in either the local or remote commitment transaction`?
Does that correctly account for the case you're mentioning?

Reply:
I applied that change, let me know if you like the new wording.